lib/msal-core/src/ErrorMessage.ts

```diff
@@ -21,6 +21,7 @@
   * OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   */
```

Comment:
This will conflict with Prithvi's error PR, so you may want to remove this change.

Reply:
I am coordinating with him for the merge, there will be conflicts in UserAgentApplication.ts too and we will handle this at the same time.